lib/generate/Request.js

```diff
@@ -62,6 +62,9 @@ function data (request, feature) {
     case BodyMode.urlencoded:
       dataUrl(body, feature)
       return
+    case BodyMode.graphql:
+      dataRaw(body, feature)
```

Comment:
This `dataRaw` procedure accesses `body.raw`. The other ones go for different fields `body.formdata` etc. Does this need to be accessing `body.graphql` or something like that?

Reply:
Yeah the new body reference should defined in `body.graphql` and I went ahead and created a new method for handling if the graphql body is set properly. Updated my test case to have a single GET and POST for graphql entries to validate both use cases.